Jenkinsfile

```diff
@@ -12,8 +12,8 @@ pipeline {
     KINTO_DOTENV = credentials('KINTO_DOTENV')
   }
   triggers {
-    pollSCM('H/5 * * * *')
-    cron('H H * * *')
+    pollSCM(env.BRANCH_NAME == 'master' ? 'H/5 * * * *' : '')
+    cron(env.BRANCH_NAME == 'master' ? 'H * * * *' : '')
```

Comment:
Did you mean to change this from daily to hourly?

Reply:
No, thanks for spotting that, and sorry for missing it!